Basic-Car-Maintenance-UITests/BasicCarMaintenanceUITests.swift

```diff
@@ -5,22 +5,39 @@
 //  Created by Mikaela Caron on 8/11/23.
 //
 
+
 import XCTest
 
 final class BasicCarMaintenanceUITests: XCTestCase {
-
-    override func setUpWithError() throws {
+    
+    let app = XCUIApplication()
+    
+    override func setUp() {
         continueAfterFailure = false
-    }
-
-    override func tearDownWithError() throws {
-    }
-
-    func testExample() throws {
-        // UI tests must launch the application that they test.
-        let app = XCUIApplication()
         app.launch()
-
-        // Use XCTAssert and related functions to verify your tests produce the correct results.
     }
+    
+    func testAddVehicle() {
+        // Navigate to the AddVehicleView
+        app.buttons["Settings"].tap()
+        XCUIApplication().collectionViews/*@START_MENU_TOKEN@*/.buttons["Add Vehicle"]/*[[".cells.buttons[\"Add Vehicle\"]",".buttons[\"Add Vehicle\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
```

Comment:
this `/*@START_MENU_TOKEN@*/` stuff shouldn't be here. This is because you had Xcode auto fill something, but you never accepted it, look how it's "highlighted" you need to either double click on it, or click on it once and press enter to accept that as the code

Reply:
Also note you defined an `app` property in the class, yet here you're redefining using a new variable for XCUIApplication

Reply:
This test only works when there's one vehicle and that the `Add Vehicle` button is in the view immediately and you don't need to scroll to tap on it